packages/editor/src/components/inputs/Vector2Input.tsx

```diff
@@ -93,7 +93,14 @@ export const Vector2Input = ({ value, onChange, onRelease, uniformScaling, hideL
         onRelease={onReleaseX}
         prefix={
           hideLabels ? null : (
-            <Vector3Scrubber {...rest} tag="div" value={vx} onChange={onChangeX} onPointerUp={onRelease} axis="x">
+            <Vector3Scrubber
+              {...rest}
+              tag="div"
+              value={vx}
+              onChange={onChangeX}
+              onPointerUp={(ev) => onReleaseX(value.x)}
```

Comment:
this seems different to what is happening in the Vector3Input - if that one works, i wonder why this doesnt. we should try to standardize these

Reply:
It doesn't work :) I'll fix that too

Reply:
fixed that as well